ruby_on_rails-postgres/README.md

```diff
@@ -0,0 +1,95 @@
+## Compose sample application
+### Rails application in dev mode
+
+Project structure:
+```
+.
+├── app
+├── docker-compose.yml
+├── Dockerfile
+├── Gemfile
+├── othres dir and files
+```
+
+[_docker-compose.yml_](docker-compose.yml)
+```
+version: '2'
```

Comment:
Please use the `3.7`  Compose specification for the Compose file

Reply:
Isn't 3 meant for swarm and 2 for docker-compose?

Reply:
No, the version 3 is cross-compatible between compose and swarm and it's the recommended version
https://github.com/docker/docker.github.io/blob/master/compose/compose-file/compose-versioning.md
https://www.compose-spec.io/

Reply:
I didn't know the Docker versions

Reply:
I fixed the error